sdk/python/kfp/dsl/metrics.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Metric classes in MLMD ontology for KFP SDK."""
+
+from typing import Any, Dict, List
+
+from kfp.dsl import artifact
+
+class Metrics(artifact.Artifact):
+  """Supports capturing summary metrics for a model from components such as ModelEvaluation."""
```

Comment:
nit: this is very optional, if not too much trouble, can you please make sure the line doesn't exceed 80 characters?